tests/unit/api/test_schemas.py

```diff
@@ -59,6 +59,21 @@ def test_environment_schema_dump_sets_api_key_in_context(django_environment):
     assert schema.context.get("environment_api_key") == django_environment.api_key
 
 
+def test_environment_schema_dump_with_hide_disabled(django_environment):
+    django_environment.project.hide_disabled_flags = True
+
+    # Given
+    schema = DjangoEnvironmentSchema()
+
+    # When
+    environment_data = schema.dump(django_environment)
+
+    # Then - disabled feature state is not present
```

Comment:
Doesn't this mean that the disabled flags won't be stored in dynamo when we write it from the core API as we discussed this morning?

Reply:
Yeah, we will not store them for environment feature states, but we will store them for identity and segments

Reply:
Thinking about it though - is that the right idea? Sorry, I know we talked about this before but I'm not sure it makes sense to _not_ store the values in dynamodb? We should just make sure that they are filtered out when serving the environment document (in core and edge), right?

Reply:
The only good thing about that is that it makes the document a little lighter

Reply:
Yeah, I guess that's a fair point. But I am definitely on the side of storing it in Dynamo too. Are you ok to update it for that?

Reply:
Yep, done